config/dev.js

```diff
@@ -21,11 +20,9 @@ module.exports = {
         'data:',
       ],
       scriptSrc: [
-        "'self'",
         amoCDN,
       ],
       styleSrc: [
-        "'self'",
         amoCDN,
       ],
```

Comment:
No not for a default app. The disco specific configs should provide it for the disco app.

Basically I'm working on the default of locking things down tightly and opening up what